readme.md

```diff
@@ -12,7 +12,25 @@ This module will extract the YouTube, Vimeo, Vine, or VideoPress Video ID from a
 $ npm install --save get-video-id
 ```
 
+
 ## Usage
+You can use right in the browser or NodeJS using ESModules or CommonJS syntax.
+
+```js
+const getVideoId = require('get-video-id');
+```
+
+or
+
+```js
+import getVideoId from 'get-video-id';
+```
+
+or
+
+```html
+<script src="getVideoId.browser.min.js"><script>
```

Comment:
Given the current build configuration, would the `src` have to be `dist/getVideoId.browser.min.js` in order for this to work? Because the rollup process does not drop this minified file into the root directory

Reply:
Updated the docs.